core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pwr.c

```diff
@@ -63,11 +63,12 @@ void stm32mp1_pwr_regulator_set_state(enum pwr_regulator id, bool enable)
 	assert(id < PWR_REGU_COUNT);
 
 	if (enable) {
-		uint64_t to = timeout_init_us(10 * 1000);
 		uint32_t ready_mask = pwr_regulators[id].cr3_ready_mask;
+		uint64_t to = 0;
 
 		io_setbits32(cr3, enable_mask);
 
+		to = timeout_init_us(10 * 1000);
```

Comment:
Wouldn't it be nice to have an explicit define for 10 * 1000? Anyway, your patch is a fix, so even without it being addressed:

`Acked-by: Gatien Chevallier <gatien.chevallier@foss.st.com>`

Reply:
I've applied your review tag and will consider defining a macro later, if you don't mind.